docs/md/modules-covid.md

```diff
@@ -0,0 +1,398 @@
+            "table" : "flowcells",
+            "column" : "last_modified_date",
+            "snapshotReaders" : [ 
+                "workflow-launcher-dev@firecloud.org" 
+            ],
+            "name" : "Terra DataRepo"
+        },
+        "commit" : "73a852f36483818a9c89cc684448a360bc642fcd",
+        "uuid" : "fb06bcf3-bc10-471b-a309-b2f99e4f5a67",
+        "executor" : {
+            "workspace" : "wfl-dev/CDC_Viral_Sequencing _ranthony_bashing_copy",
+            "methodConfiguration" : "wfl-dev/sarscov2_illumina_full",
+            "methodConfigurationVersion" : 2,
+            "fromSource" : "importSnapshot",
+            "name" : "Terra"
+        },
+        "version" : "0.8.0",
+        "sink" : {
+            "workspace" : "wfl-dev/CDC_Viral_Sequencing _ranthony_bashing_copy",
+            "entityType" : "flowcell",
+            "fromOutputs" : {
+                ...
+            },
+            "identifier" : "run_id",
+            "name" : "Terra Workspace"
+            }
+        }
+    ```
+
+**POST /api/v1/stop**
+
+=== "Sample Request"
+
+    ```
+    curl -X POST 'http://localhost:8080/api/v1/start' \
+         -H 'Authorization: Bearer '$(gcloud auth print-access-token) \
+         -H 'Accept: application/json' \
+         -H 'Content-Type: application/json' \
+         -d $'{ "uuid": "fb06bcf3-bc10-471b-a309-b2f99e4f5a67" }'
+    ```
+
+=== "Sample Response"
+
+    ```
+    {
+        "started" : "2021-06-15T18:54:57Z",
+        "watchers" : [ 
+            "tester@123.com" 
+        ],
+        "labels" : [ 
+            "hornet:test", 
+            "project:wfl-dev/CDC_Viral_Sequencing _ranthony_bashing_copy" 
+        ],
+        "creator" : "ranthony@broadinstitute.org",
+        "updated" : "2021-06-15T18:59:28Z",
+        "created" : "2021-06-15T18:50:52Z",
+        "source" : {
+            "dataset" : "4bb51d98-b4aa-4c72-b76a-1a96a2ee3057",
+            "table" : "flowcells",
+            "column" : "last_modified_date",
+            "snapshotReaders" : [ 
+                "workflow-launcher-dev@firecloud.org" 
+            ],
+            "name" : "Terra DataRepo"
+        },
+        "stopped" : "2021-06-15T18:59:28Z",
+        "commit" : "73a852f36483818a9c89cc684448a360bc642fcd",
+        "uuid" : "fb06bcf3-bc10-471b-a309-b2f99e4f5a67",
+        "executor" : {
+            "workspace" : "wfl-dev/CDC_Viral_Sequencing _ranthony_bashing_copy",
+            "methodConfiguration" : "wfl-dev/sarscov2_illumina_full",
+            "methodConfigurationVersion" : 2,
+            "fromSource" : "importSnapshot",
+            "name" : "Terra"
+        },
+        "version" : "0.8.0",
+        "sink" : {
+            "workspace" : "wfl-dev/CDC_Viral_Sequencing _ranthony_bashing_copy",
+            "entityType" : "flowcell",
+            "fromOutputs" : {
+                ...
+            },
+            "identifier" : "run_id",
+            "name" : "Terra Workspace"
+            }
+        }
+    ```
```

Comment:
The example responses swamp the requests, which are more interesting here, I believe.
Maybe emphasize that the responses depend on the kind of workload requested?
There is trailing whitespace in this file if you care.

Reply:
Removed the responses